build.cake

```diff
@@ -133,7 +133,31 @@ Task("PublishCalamariProjects")
         }
 });
 
+Task("PublishSashimiTestProjects")
+    .IsDependentOn("Build")
+    .Does(() => {
+        var projects = GetFiles("./source/**/Sashimi.*.Tests.csproj");
+		foreach(var project in projects)
+        {
+            var sashimiFlavour = project.GetFilenameWithoutExtension().ToString();
+
+                void RunPublish() {
+                     DotNetCorePublish(project.FullPath, new DotNetCorePublishSettings
+		    	    {
+		    	    	Configuration = configuration,
+                        OutputDirectory = $"{publishDir}/{sashimiFlavour}"
+		    	    });
+                }
+
+                RunPublish();
+
+            Console.WriteLine($"{publishDir}/{sashimiFlavour}");
```

Comment:
is this for debugging?

Reply:
I copied it from the `PublishCalamariProjects` task, which had it. https://github.com/OctopusDeploy/Sashimi.Terraform/pull/3/files#diff-4d3634747586b964df9f7712bc3ec669R131

Reply:
That is wrong. You should be using the cake syntax, see https://cakebuild.net/dsl/logging/

Reply:
ah ok, I'll also fix it on Sashimi then